### ISSUE TITLE
Israeli F-15A Baz with Ophir  Longrange oblique photography (LOROP) Recce pod

### ISSUE THREAD

**Issue by Huib18** [PygmalionOfCyprus/cmo-db-requests#4122]
### DB Selector

DB3K

### User(s)

Aircraft Israel F15A

### Weapons Carried

4x Pyton 5 IR missiles
3x AIM-120C7 AMRAAM or AIM-7F SPARROW
1x ECM pod EL/L-8202
Conformal fuel tanks
Ophir LOROP reconnaissance pod

### Requires Buddy Illumination

No

### Default Combat Radius (nm)

Same as A/A Heavy + range from conformal fuel tanks

### Weather Restrictions

All-Weather

### Time-of-Day Restrictions

Day & Night

### Capable of Quick Turnaround

Yes

### Comments

**You can probably use  Weapon Condor2 recce pod in the DB #3671**

Seen with AIM-7 Sparrow and AIM-120 AMRAAMs.
Datalink with ground station.
From the start of the F-15A era till current time because the pod transfered from RF-4E Kurnas

HAIFA, Israel – [Elbit Systems](https://www.intelligent-aerospace.com/paris-air-show/article/14034743/elbit-systems-hermes-45-will-debut-at-paris-airshow) launched CONDOR MS, a new long range oblique photography (LOROP) system that introduces multi-spectral (MS) sensing capability and [artificial intelligence (AI)](https://www.intelligent-aerospace.com/avionics/article/16545234/artificial-intelligence-ais-increasing-role-in-autonomous-aircraft-technology) analytics to stand-off strategic intelligence gathering missions.

CONDOR MS integrates three high resolution electro-optic (EO) sensors into the company’s certified and widely operational CONDOR2 system: visible & near infra-red (VNIR), medium-wave infrared (MWIR), and short-wave infrared (SWIR).

(Although not the same but probably the same capabilities:
Same size as the 16 gallon backup fuel tank used in F-300’s and of the pods that have the size, Condor II LoROP is simultaneously in the EO and IR bands a discovery with built-in INS/GPS positioning system capable of shooting system. The system, given the approximate range of 100km, obtained the images Rafael production takes place in real time with the DLV-52 Ku band data link system he can transfer it to the station or record it on himself. Data link the performance of the system is 320Mb/s at a range of 120km and the service altitude is maximum 19,800m.
[https://www.siyahgribeyaz.com/2011/09/kopan-turkiye-israil-iliskilerinin.html](url)

### Sources

[https://www.hlj.com/1-32-scale-ophir-reconnaissance-pod-for-iaf-re-4e-kurnass-and-f-15-baz-isr32031](url)
[https://www.facebook.com/isradecal/posts/isracast-ophir-recce-pod-for-iaf-f-15-baz-133-squadron-and-rf-4e-kurnass-69-and-/2463405893804026/](url) (8x pictures of RF-4 and F-15A)
[https://pbs.twimg.com/media/GA6jcwPXkAAVkWK?format=jpg&name=large](url)
[https://jp.super-hobby.com/zdjecia/0/5/9/52658_f3.jpg](url)
[https://www.google.com/url?sa=i&url=https%3A%2F%2Fwww.keymilitary.com%2Farticle%2Feagle-full-throttle&psig=AOvVaw1ukE2Fsebbi9fAXTgZdsbm&ust=1704400221329000&source=images&cd=vfe&ved=0CBMQjRxqFwoTCMjwuuCHwoMDFQAAAAAdAAAAABAE](url)
[
![image](https://github.com/PygmalionOfCyprus/cmo-db-requests/assets/60408110/b2607f66-ece2-47ca-afd2-bbb90d371920)
](url)

**Comment by bostonmyk** [comment]
Here's a F-15D

https://x.com/GuyPlopsky/status/1815014257882497256

**Comment by Huib18** [comment]
Good catch bostonmyk, updated the title and the description to add the F-15D
I guess the F-15C is also capable but need evidence... Keep your eyes open pls ;-)
